content-repo/extra-docs/packs/Rapid-Breach-Response.md

```diff
@@ -0,0 +1,133 @@
+---
+id: Rapid Breach Response
+title: Rapid Breach Response Layout
+description: Analyzing cyber attacks risk is a vital component of managing and remediating security events. The Rapid Breach Response Layout content pack enables security teams to automate and streamline cyber attacks risk analysis with dedicated playbooks for each scenario.
+ 
+---
+This content pack enables security teams to quickly evaluate cyber attack risk using dedicated playbooks for each scenario, such as Hafnium - Exchange 0-day Exploits, SolarStorm, and PrintNightmare.
+
+## Pack Requirements
+You must install and configure the **Demisto Rest API** before you start using this pack.
+
+### Demisto REST API Integration
+The IR Tracking dynamic section is built using the server Rest API and is vital for layout functionality. The scripts require that you install the **Demisto REST API** integration and configure an integration instance.
+
+1. In Cortex XSOAR, go to **Settings** > **INTEGRATIONS** > **API Keys**.
+2. Click **Get Your Key**, enter a name for the API key, and click **Generate Key**.
+3. **(IMPORTANT)** Copy and save the API key, you will not be able to access it again.
+4. Go to **Settings** > **INTEGRATIONS** > **Servers & Services** and search for **Demisto REST API**.<br/>
+    Note: <br/>
+    In multi-tenant environments the **Demisto REST API** integration should be configured at the parent level and propagated down to the child.
+5. Click **Add instance** and enter the required information.
+    - A meaningful name for the integration instance
+    - The Demisto server URL
+    - The API key you generated
+7. Click **Test** to make sure that that server and API key are reachable and valid.
+8. Click **Save & exit**.
+
+## Pack Workflow
+### Collect indicators.
+The first step in most Rapid Breach Response playbooks is collecting indicators. Indicator sources can be blogs, advisories, or any other highly reliable reference. Indicator format can be text, rules, and signatures.
+### Extract, create, and tag the indicators.
+After completing data collection and indicator extraction, execute the new **Set RapidBreachResponse Incident Info** playbook, which provides:
+- Links to sites from which the data was collected
+- The playbook description
+- The total number of indicators extracted
+### Execute threat hunting.
+After completing information collection and processing, execute threat hunting to find indicators of compromise (IOC) related to the attack in the customer organization. This includes the following tasks:
+- PANW Hunting
+- Cortex XDR signatures and XQL hunting
+- SIEM hunting
+- Advanced Hunting based on attack patterns
+- IOC hunt using Endpoint Detection and Response
+
```

Comment:
lets add also the steps we are doing in the remediation, eradication

Reply:
Done